post_info.php

```diff
@@ -242,20 +242,49 @@ function handle_courses_sections($association_list, $action)
                 $bind_variables["course_code"] = $course_code;
             }
         } elseif ($action == "add_or_update") {
-            $bind_variables["course_code"] = $course_code;
-            $bind_variables["course_title"] = $course_title;
-            $bind_variables["department_name"] = $department_name;
-            $bind_variables["section_code"] = $section["section_code"];
-            $bind_variables["term"] = $term;
+            $bind_variables[0] = array(
+                "course_code" => $course_code,
+                "course_title" => $course_title,
+                "department_name" => $department_name
+            );
+            $bind_variables[1] = array(
+                "course_code" => $course_code,
+                "term" => $term,
+                "section_code" => $section["section_code"]
+            );
             if ($section["section_id"] != null) {
-                $bind_variables["section_id"] = $section["section_id"];
+                $bind_variables[1]["section_id"] = $section["section_id"];
             }
+            $bind_variables[2] = null;
```

Comment:
You didn't answer my question. Where is `$bind_variables[2]` used?

Reply:
`$bind_variables[2]` is used on line 272.